src/Resources/skeleton/authenticator/login_form.tpl.php

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Log in!{% endblock %}
+
+{% block body %}
+<form method="post">
+    {% if error %}
+        <div class="alert alert-danger">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+    {% endif %}
+
+    <h1 class="h3 mb-3 font-weight-normal">Please sign in</h1>
+    <label for="inputEmail" class="sr-only">Email address</label>
+    <input type="email" value="{{ last_username }}" name="<?= $username_field; ?>" id="inputEmail" class="form-control" placeholder="Email address" required autofocus>
+    <label for="inputPassword" class="sr-only">Password</label>
+    <input type="password" name="password" id="inputPassword" class="form-control" placeholder="Password" required>
```

Comment:
Ok, if this is the markup I gave you, then it's based off of this - https://getbootstrap.com/docs/4.1/examples/sign-in/ - which actually includes an extra CSS File to get some of the magic. But, I think it still looks fairly close to good, and without the extra markup. So, let's keep it for now.